website/components/live-example/Knobs.tsx

```diff
@@ -217,27 +209,24 @@ function Select({
   prop,
   options,
   darkMode,
-}: BasicSelectInterface | GlyphSelectInterface) {
+}: BasicSelectInterface) {
   const labelId = useMemo(() => selectIdAllocator.generate(), []);
 
-  const handleChange = ({ target }: React.ChangeEvent<HTMLSelectElement>) => {
-    onChange(target.value, prop);
+  const handleChange = (value: string) => {
+    onChange(value, prop);
   };
 
   const generateOptionsCallback = () => {
     return options
       .sort((a, b) => a.localeCompare(b))
```

Comment:
why not provide the options in the order given?

Reply:
No reason not to